### ISSUE TITLE
got ERR: no packets received recently (maybe there's a firewall/NAT in between)

### ISSUE THREAD

**Issue by superyhee** [aler9/rtsp-simple-server#93]
## Which version are you using?

v0.9.15

## Which operating system are you using?

 Linux

## Question
I run rtsp-simple-server on aws ec2 ,but when i push my webcam rtsp stream to remote server with the command: ffmpeg -f avfoundation  -framerate 30 -i "0" -f rtsp rtsp://ip:8554/ipc
I got the following error on server side:
2020/09/29 06:51:04 [2/1/1] [client 54.222.45.4:14387] ERR: no packets received recently (maybe there's a firewall/NAT in be tween)
I got following errors on client side :
av_interleaved_write_frame(): Broken pipe
 How to resolve it?

**Comment by aler9** [comment]
Hello, try switching to the TCP protocol, which uses the same connection of the handshake, while UDP requires two new UDP connections that are often blocked by firewalls:
```
ffmpeg -rtsp_transport tcp -f avfoundation -framerate 30 -i "0" -f rtsp rtsp://ip:8554/ipc
```

**Comment by nfynt** [comment]
Hi @aler9 

I am using FFMPEG build from 2020 compiled with gcc 10.2, It doesn't include option for rtsp_transport or avfoundation! Is there a workaround or some specific version for this?

Thanks

**Comment by aler9** [comment]
> It doesn't include option for rtsp_transport or avfoundation! Is there a workaround or some specific version for this?

yes, look at the readme for instructions on how to publish a webcam:

https://github.com/aler9/rtsp-simple-server#serve-a-webcam

**Comment by MrDredD** [comment]
Hello @aler9 
Thanks for a wonderful service!

# **Version and operating system**
v0.14.0, Fedora 33 and  Ubuntu 18.04

# Question 
When I'm trying to proxy RTSP stream, I'm getting the same error repeatedly. Stream stops and then reconnects. It works very unstable, but still I'm able to read proxied stream. I tried to remove udp from the list in config, but it had no affect
Is there is a way to overcome this problem?
